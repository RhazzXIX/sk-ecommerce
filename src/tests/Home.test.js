import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import Home from "../app/page";

describe("Home component", () => {
  it("Renders on screen", () => {
    const { container } = render(<Home />);

    expect(container).toMatchSnapshot();
  });
});
