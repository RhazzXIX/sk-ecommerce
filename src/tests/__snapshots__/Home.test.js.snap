// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component Renders on screen 1`] = `
<div>
  <main
    class="main"
  >
    <h1>
      SK Innovation Storage
    </h1>
    <a
      href="/list"
    >
       Virtual List storage
    </a>
  </main>
</div>
`;
